### ISSUE TITLE
Node.js version deprecated

### ISSUE THREAD

**Issue by kthy** [extractions/setup-just#13]
Now receiving the following warning in my workflows:

```text
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: extractions/setup-just@v1.
For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

**Comment by kevinmatthes** [comment]
One option might be to migrate to this Action:  https://github.com/taiki-e/install-action.  It is highly actively maintained and offers not only Just but also a set of other extremely useful Rust binaries.

I know that this suggestion does not solve the reported problem but it is a possibility to avoid further warnings, after all.  Furthermore, GitHub has also automatically migrated old Node 12 Actions to Node 16, in the past, thus, it might be possible that some similar service is provided, again, at some point of time.

**Comment by epicserve** [comment]
Any updates on this?

**Comment by epicserve** [comment]
I [fixed it](https://github.com/epicserve/django-base-site/commit/d0ed5c0f9f5e62ae5a82e0216a32c9531bf79836) for now by just installing via bash.

```
- name: Use Just for running commands
   run: curl --proto '=https' --tlsv1.2 -sSf https://just.systems/install.sh | bash -s -- --to /usr/local/bin
```

**Comment by kthy** [comment]
Fixed by fbd91a81bd6f8bbd62cc6c9e2e8540bac23b83c9.
